www/viewgame

```diff
@@ -710,6 +710,22 @@ $extraHead = "<script src=\"xmlreq.js\"></script>";
 if ($should_hide) {
     $extraHead .= "<meta name=\"robots\" content=\"noindex\">";
 }
+
+// add Open Graph elements
+$extraHead .= "<meta property=\"og:site_name\" content=\"IFDB\" />";
+$extraHead .= "<meta property=\"og:type\" content=\"website\" />";
+$extraHead .= "<meta property=\"og:url\" content=\"";
+$extraHead .= get_root_url() . "viewgame?id=$id\" />";
+$extraHead .= "<meta property=\"og:title\" content=\"$title\" />";
+if(!isEmpty($desc)) {
+    $extraHead .= "<meta property=\"og:description\" content=\"$desc\" />";
+}
+if ($hasart) {
+    $extraHead .= "<meta property=\"og:image\" content=\"";
+    $extraHead .= get_root_url() . "viewgame?coverart&amp;id=$id";
+    $extraHead .= "&amp;thumbnail=175x175\"/>";
```

Comment:
```suggestion
    $extraHead .= get_root_url() . "viewgame?coverart&amp;id=$id\"/>";
```

I don't think we want to use a small thumbnail for this; we should use the biggest size we have. Stuff I've read online indicates that really big images (1200x1200) are good. https://iamturns.com/open-graph-image-size/